src/Protocol/Diaspora.php

```diff
@@ -3579,11 +3608,11 @@ private static function constructAttend(array $item, array $owner)
 				return false;
 		}
 
-		return(["author" => self::myHandle($owner),
+		return ["author" => self::myHandle($owner),
```

Comment:
Please re-indent this array and replace quotes.
```suggestion
		return [
			'author' => self::myHandle($owner),
```